documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -165,11 +176,8 @@ Hibernate OGM allows to specify a TTL value on entities and associations. The TT
 the entity using the http://redis.io/commands/pexpire[PEXPIRE] command. Every write to Redis will
 set a new TTL.
 
-The annotation on the entity level expresses that all associations of the `Zoo`
-class should be stored in separate assocation documents.
-This setting applies to the `animals` and `employees` associations.
-Only the elements of the `visitors` association will be stored in the document of the corresponding `Zoo` entity
-as per the configuration of that specific property which takes precedence over the entity-level configuration.
+You can specify a different TTL on associations if they are stored as separate documents. In the case of `visitors`
```

Comment:
@gunnarmorling Found one during my sports session :). It might be useful if you consider a list of users visiting a page or a product, or the list of active users on a forum topic.

Redis is typically used for this sort of transient information.